.github/workflows/canary-release.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     uses: ./.github/workflows/release.yml
     secrets: inherit
     with:
+      profile: canary
```

Comment:
Should this be enabled for dev releases as well? If so, the option should be more generic (i.e. a profiling build)